src/Catty/Extension&Delegate&Protocol/Extensions/UIViewController/ProjectDetailStoreViewControllerExtension.swift

```diff
@@ -72,4 +72,51 @@ extension ProjectDetailStoreViewController {
         }
         return InputValidationResult.validInput()
     }
+
+    func download(name: String) {
```

Comment:
When re-downloading a project which already exists on the device, the project gets overridden. Maybe that is the case because the `name` parameter here is never used?

Reply:
Yes, you a right! I did set the name of the current project and not the given name! Thank you for noticing! I've changed that too.